src/js/tabs/osd.js

```diff
@@ -565,7 +581,13 @@ OSD.loadDisplayFields = function() {
             defaultPosition: 1,
             draw_order: 120,
             positionable: true,
-            preview: 'R:2:200:P',
+            variants: [
+                'osdTextElementVTXchannelVariant-full',
+                'osdTextElementVTXchannelVariant-power',
```

Comment:
Sorry to not notice that the first time...
The other "variants" have not a '-' between the name and the variant. Is good to continue with the same nomenclature and not merge ways.
Another option is to replace all the others, but is simpler to modify only this.

Reply:
all fixed.

thanks for feedback